### PR TITLE
Implement RegEnvVarList()

### PR DESCRIPTION
**Pull request by shubhsboson**
* implement wslpy.system;
     - [x] implement registry(input, key);
     - [x] implement RegEnvVarList() ~~using **regInfoFetch**(input, key);~~
     - [x] implement regEnvVar(input) to get environment variables via Registry.
#Doesn't actually use regInfoFetch(input,key)

### CODE REVIEW

File: wslpy/system.py [hunk 1]
```diff
@@ -30,13 +30,26 @@ def __shellEnvVar__(regname):
         print(err)
 
 
+def _regEnvVarList_():
```
Comment (patrick330602) [inline wslpy/system.py:33]: What you are doing here is getting the System Environment Varibale; What we want to archive here is to get the user variables from the registry such as users Desktop folder, Documents folder, etc, from registry `HKCU:\Software\Microsoft\Windows\CurrentVersion\Explorer\User Shell Folders`.

**Comment by patrick330602** [comment]
My mistake, it is the correct implementation. But still need to change the name.

File: wslpy/system.py (hunk 1 above)
Comment (patrick330602) [inline wslpy/system.py:33]: Rename this to __sysEnvVarList__()

File: wslpy/system.py [hunk 2]
```diff
@@ -77,5 +90,16 @@ def shellEnvVar(input):
     return __shellEnvVar__(input)
 
 
+def RegEnvVarList():
```
Comment (patrick330602) [inline wslpy/system.py:93]: Rename this to sysEnvVarList, and update the documentation

File: wslpy/system.py (hunk 1 above)
Comment (shubhsboson) [reply wslpy/system.py:33]: Done

File: wslpy/system.py (hunk 2 above)
Comment (shubhsboson) [reply wslpy/system.py:93]: Done

**Comment by shubhsboson** [comment]
So I've implemented registry(input,key) .

**Comment by patrick330602** [comment]
Hi,

Sorry to be late, I will try my best to spare some time to review the code tonight.

**Pull request merged**